### PR TITLE
doc: releases/luminous: release notes for 12.2.12

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/luminous.rst [hunk 1]
```diff
@@ -1,6 +1,103 @@
 v12.2.11 Luminous
```
Comment (liewegas) [inline doc/releases/luminous.rst]: should be 12.2.12

File: doc/releases/luminous.rst [hunk 2]
```diff
@@ -1,6 +1,103 @@
...
+This is the twelfth bug fix release of the Luminous v12.2.x long term stable
+release series. We recommend that all users upgrade to this release.
+
+Notable Changes
+---------------
+
+* In 12.2.9 and earlier releases, keyring caps were not checked for validity,
+  so the caps string could be anything. As of 12.2.10, caps strings are
+  validated and providing a keyring with an invalid caps string to, e.g.,
+  `ceph auth add` will result in an error.
```
Comment (smithfarm) [inline doc/releases/luminous.rst]: I believe this release note was written with the assumption that the PR https://github.com/ceph/ceph/pull/24906 would be merged for the v12.2.10 release, but in reality it was merged for the v12.2.12 release so I think the release numbers should be changed accordingly.

@tchaikov Do you agree?

Comment (tchaikov) [reply doc/releases/luminous.rst]: @smithfarm thanks for pointing this out. yeah, we should s/12.2.9/12.2.11/ here.

Comment (theanalyst) [reply doc/releases/luminous.rst]: good catch, updating

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by theanalyst** [comment]
updated changelog to add issues without the tracker

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by theanalyst** [comment]
jenkins render docs

**Pull request merged**